### PR TITLE
Flag not available partners (T202841)

### PR DESCRIPTION
**Pull request by uyscuti-wiki**
Browse page now visually indicates not available partners

### CODE REVIEW

**Comment by Samwalton9** [comment]
models.py still has the suggest code :D

File: TWLight/static/css/local-rtl.css [hunk 1]
```diff
@@ -331,6 +331,14 @@ div.panel-body form.form-inline input#submit-id-submit.btn.btn-default {
   padding: 6px 25px 0px 0px;
 }
 
+.resource-label-danger {
+  background-image: url("../img/bullet-danger.png");
+  background-repeat: no-repeat;
+  background-position: top left; /* Must set explicitly for CSS Janus. */
```
Comment (Samwalton9) [inline TWLight/static/css/local-rtl.css]: Looks like this should have a background-position of top right, since this is RTL (see L329)

**Comment by Samwalton9** [comment]
The Travis build fail can be ignored, it's unrelated to your PR.

**Comment by uyscuti-wiki** [comment]
> models.py still has the suggest code :D

ugh, a pain!

**Comment by jsnshrmn** [comment]
@Samwalton9 can you do a quick re-review since you requested changes? Basically, you approve the changes from @uyscuti-wiki since your review to indicate that he successfully made the requested changes.

**Pull request merged**